xrrandom/distributions.py

```diff
@@ -0,0 +1,55 @@
+import scipy.stats as stats
+
+from .scipy_stats_sampling import virtually_sample_distribution
+
+class VirtualDistribution:
+    """Representation of scipy.stats distribution using virtual samples.
+    
+    The location (``loc``) keyword specifies the mean.
+    The scale (``scale``) keyword specifies the standard deviation
+    Other shape parameters specific for the distribution are passed to 
+    the scipy distribution.
+
+    The size of the output array is determined by broadcasting the shapes of the
+    input arrays, scipy.stats ``shape`` parameter is ignored.
+
+    The ``samples`` parameter determines how many random samples will be drawn,
+    can be changed later using ``xrrandom.sample`` or ``xrrandom.change_virtual_samples``.
+
+    For the virtual sampling idea look at :py:func`xrrandom.sampling.generate_virtual_samples``
+    """   
+
+    def __init__(self, distr):
+        self._distr = distr
```

Comment:
could be used in `__repr__`

Reply:
I've added `__repr__` for frozen distributions, the default representation seems ok for the other classes.